sql/mysqld.cc

```diff
@@ -444,6 +444,9 @@ ulong log_warnings;
 uint host_cache_size;
 ulonglong tmp_table_rpl_max_file_size;
 
+my_bool log_legacy_user;
+char *opt_legacy_user_identifier;
```

Comment:
Does MySQL allow you to mark this buffer as `const char *`?  If so, that would be preferable.